src/common/transformations/include/transformations/common_optimizations/loop_inputs.hpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+#include <vector>
+
+#include "openvino/pass/graph_rewrite.hpp"
+#include "transformations_visibility.hpp"
+
+namespace ov {
+namespace pass {
+
+class TRANSFORMATIONS_API LoopInputs;
```

Comment:
LoopInputs does not reflect information on modifications performed in the transformation
Please rename it